subsys/bluetooth/services/fast_pair/fp_storage/Kconfig.fp_storage

```diff
@@ -34,6 +34,15 @@ config BT_FAST_PAIR_STORAGE_OWNER_ACCOUNT_KEY
 	help
 	  Enable support for the Owner Account Key in the Storage module.
 
+config BT_FAST_PAIR_STORAGE_BOND_MANAGEMENT
+	bool
+	depends on BT
```

Comment:
I think we can remove the dependency here. The Bluetooth address header does not count as it is always included regardless of the `BT` Kconfig option value

Reply:
We have following dependency:
```
#if CONFIG_BT_FAST_PAIR_STORAGE_BOND_MANAGEMENT
#define FP_BONDS_ARRAY_LEN	(CONFIG_BT_MAX_PAIRED + CONFIG_BT_MAX_CONN)
#else
#define FP_BONDS_ARRAY_LEN	0
#endif
```
If we enable the `CONFIG_BT_FAST_PAIR_STORAGE_BOND_MANAGEMENT` without `BT`, it will lead to build failure.

Reply:
Are the build failures related to these Kconfigs or to some other missing C definitions?